README.md

```diff
@@ -30,8 +30,10 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221179256-fd22edc2-f2a3-4d24-9a5d-870d8eef5dd1.png)

**[Optional]**

- [ ] It would be nice if you add the following commands in their respective section to make your readme file descriptive :+1:

## Setup

```
git clone https://github.com/habasm/Math-Magicians.git
cd Math-Magicians
```

## Install

```
npm install
```

## Usage

```
npm start
```

## Test

```
npm test
```